main.py

```diff
@@ -5,25 +5,59 @@
 import win32con
 import json
 from threading import Thread
-
+from screeninfo import get_monitors
 
 user32 = ctypes.windll.user32
 user32.SetProcessDPIAware()
 
 screen_width = user32.GetSystemMetrics(0)
 screen_height = user32.GetSystemMetrics(1)
 windowList = []
-saveList = []
+saveList = {}
 selected_app = "0"
 temp_win_height = 1080
 temp_win_width = 1920
+monitors = {}
+selected_monitor = None
+
+# Get monitor info from screeninfo
+for index, m in enumerate(get_monitors()):
+    # I'm not using m.name here because mine were really weird, like m.name='\\\\.\\DISPLAY17',
+    # but there is probably a better way to do this
+    name = f"Display {str(index + 1)}" 
+    if(m.is_primary):
+        name += " (Primary)"
+        selected_monitor = name
+    monitors[name] = m
+
+resolution_options = {
```

Comment:
I appreciate your effort with preset resolutions, but one of the major requests from reddit was to add support for custom resolutions so I will do that here later. For now I will merge it.